.github/workflows/publish-and-deploy.yml

```diff
@@ -38,8 +38,8 @@ jobs:
       - name: Build
         uses: whoan/docker-build-with-cache-action@v5
         with:
-          context: basistheory/${{ matrix.demo }}
-          image_name: ${{ matrix.demo }}
+          context: demos/${{ matrix.demo }}
+          image_name: basistheory/${{ matrix.demo }}
```

Comment:
Is this acceptable?
```suggestion
          image_name: basistheory/demos/${{ matrix.demo }}
```

Reply:
We can try it, but i believe its `org/container:tag` in dockerhub for the syntax. All the examples I am seeing are in the previously noted format.

Reply:
```
Your repository name must contain a combination of alphanumeric characters and may contain the special characters ., _, or -. Letters must be lowercase.
```
^what dockerhub gives me if I try to do `demos/reference-check` i can rename the repository to `demo-reference-check` if you want more contextual clarity in the name / purpose of the container. There is also a description section I was going to fill out and connect everything back to the github repo.

Reply:
`reference-check-demo` should suffice. Thanks